packages/nightingale-track/src/FeatureShape.ts

```diff
@@ -304,4 +309,35 @@ export default class FeatureShape {
     }Z`;
     return rect + triangle;
   }
+
+  tag() {
+    const h = this.#ftHeight;
+    const w = this.#ftWidth;
+    const d1 = Math.sqrt(2 * Math.pow(h, 2));
```

Comment:
```suggestion
    const d1 = Math.sqrt(2) * h
```
This seems equivalent, unless we're guarding specifically against `h` < 0?